package.json

```diff
@@ -9,8 +9,8 @@
     "build": "react-scripts build",
     "build:install": "npm install && npm run lintnofix && npm run build",
     "snapshot": "react-snap",
-    "test": "react-scripts test --env=jsdom",
-    "testcafe": "testcafe -c 1 chrome testcafe/ --app \"react-scripts start\" --app-init-delay 30000",
+    "test": "testcafe -c 4 chrome:headless testcafe/ --app \"serve -s build\"",
```

Comment:
headless doesn't work on travis?

Reply:
headless mode had slightly worse or same performance on travis when I tested it.

Reply:
if it's approx the same I'd rather keep one version of the test to avoid fragmentation